pandas/tests/formats/test_printing.py

```diff
@@ -9,6 +9,17 @@
 _multiprocess_can_split_ = True
 
 
+def test_to_string_formatters_index_header():
+    from pandas import DataFrame
```

Comment:
add the issue number (this PR since there is no number) as a comment

if you need the import put at the top of the file